test/scripts/bridge/build-ethereum-node.sh

```diff
@@ -7,54 +7,54 @@ scripts_path="$(realpath ./scripts/bridge)"
 
 source $scripts_path/set-env.sh
 
-echo "Building lodestar Snowfork fork"
-
-
-set_slot_time() {
-    local new_value=$1
-    echo "Hack lodestar for faster slot time"
-    local preset_mainnet_config_file="$artifacts_dir/lodestar/packages/config/src/chainConfig/configs/mainnet.ts"
-    if [[ "$(uname)" == "Darwin" && -z "${IN_NIX_SHELL:-}" ]]; then
-        gsed -i "s/SECONDS_PER_SLOT: .*/SECONDS_PER_SLOT: $new_value,/g" $preset_mainnet_config_file
-    else
-        sed -i "s/SECONDS_PER_SLOT: .*/SECONDS_PER_SLOT: $new_value,/g" $preset_mainnet_config_file
+# Check if lodestar is already installed in our output directory
+if [ -f "$output_bin_dir/lodestar" ] && [ -x "$output_bin_dir/lodestar" ]; then
+    lodestar_version=$($output_bin_dir/lodestar --version 2>/dev/null | grep -o "v[0-9]\+\.[0-9]\+\.[0-9]\+" || echo "unknown")
+    echo "Lodestar $lodestar_version seems to be already installed in $output_bin_dir. Skipping download"
+    if [[ "$lodestar_version" != "$LODESTAR_TAG" && "$lodestar_version" != "unknown" ]]; then
+        echo "WARNING: Installed Lodestar version ($lodestar_version) does not match expected version ($LODESTAR_TAG)"
     fi
-}
-
-echo "Downloading lodestar"
-
-if [ -d "$lodestar_dir" ];
-then
-  echo "Lodestar seems to be already downloaded. Skipping downloading again"
 else
-  git clone https://github.com/ChainSafe/lodestar $lodestar_dir
-  pushd $lodestar_dir
-  git fetch && git checkout $LODESTAR_TAG
-  set_slot_time 1
-  popd
+    echo "Downloading lodestar from GitHub releases (version $LODESTAR_TAG)"
+    
+    # assuming linux since the rest of the project requires it
+    DOWNLOAD_URL="https://github.com/ChainSafe/lodestar/releases/download/$LODESTAR_TAG/lodestar-$LODESTAR_TAG-linux-amd64.tar.gz"
```

Comment:
I do not think there is a requirement for this. If possible, let's do this:
if it is linux, we can directly download it. 
otherwise, we can build it.

Reply:
done